domUpdates.js

```diff
@@ -38,6 +38,55 @@ const domUpdates = {
     wheel.wheelSpin()
     $('.hidden-guess-section').addClass('spin-the-wheel')
     $('.wheel-prize-display').text(wheel.currentValue)
+  },
+
+  appendGuessToDom(currentGuess, answerLettersArray) {
+    if (answerLettersArray.length < 15) {
+      let column = 0
+      let stringedArray = answerLettersArray.split('')
+      stringedArray.forEach((letter) => {
+        if(letter === currentGuess) {
+          var thisDiv = document.querySelector(`.tile-${column}`);
+          thisDiv.innerText = currentGuess;
+        }
+        column += 1
+      })
+    } else {
+     let column = 0
+     let columnOne = 0
+     let wordArray = answerLettersArray.split(' ');
+     wordArray.forEach((word) => {
+       column += word.length
+       if (column < 15) {
+         let newArray = word.split('')
+         newArray.forEach((letter) => {
+          if(letter === currentGuess) {
+            var thisDiv = document.querySelector(`.tile-${columnOne}`);
+            thisDiv.innerText = currentGuess;
+          }
+          columnOne += 1
+        })
+        columnOne += 1
+
+       } else {
+         column = 14
+         let newArray = word.split('')
+         newArray.forEach((letter) => {
+           console.log(letter)
+           if(letter === currentGuess) {
+            var thisDiv = document.querySelector(`.tile-${column}`);
+            thisDiv.innerText = currentGuess;
+          }
+          column += 1
+        })
+       }
+     })
+   }
+
```

Comment:
I think a good way to refactor this long method is to make sure that methods that are kept in your `domUpdates` file are only manipulating the DOM. `appendGuessToDom` should only be doing what its name state - appending the guess to the DOM. I'm a bit confused as to why this method seems to be responsible for checking/evaluating the guess against the answer as well as updating the DOM. Anything that has to do with the logic of your App (like comparing a guess to an answer) should be handled within the appropriate class.